Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1

```diff
@@ -462,3 +462,33 @@ function Restore-VmfsVolume {
 
     $Esxi | Get-VMHostStorage -RescanVMFS -ErrorAction stop | Out-Null
 }
+
+<#
+    .SYNOPSIS
+     This function removes the static iSCSI configurations from a specified Esxi Host
+
+    .PARAMETER VMHostName
+     Name of the VMHost (ESXi server)
+
+    .EXAMPLE
+     Remove-VMHostStaticiSCSITargets -VMHostName "vmhost1"
+
+    .INPUTS
+     VMHostName
+
+    .OUTPUTS
+     None
+#>
+function Remove-VMHostStaticiSCSITargets {
+    [CmdletBinding()]
+    [AVSAttribute(10, UpdatesSDDC = $false)]
+    Param (
+        [Parameter(
+                Mandatory=$true,
+                HelpMessage = 'VMost name')]
+        [ValidateNotNull()]
+        [String]
+        $VMHostName
+    )
+    Get-VMHost $VMHostName | Get-VMHostHba -Type iScsi | Get-IScsiHbaTarget | Where {$_.Type -eq "Static"} | Remove-IScsiHbaTarget -Confirm:$false
```

Comment:
It would be great to add a check at the beginning of this function to see if host specified by $VMHostName does exist, prior to calling Remove-IscsiHbaTarget

Reply:
Get-VMHost should throw an error if it fails to find the host but let me add extra check